### DISCUSSION TITLE
Add support for Windows Store submission

### DISCUSSION THREAD

**Discussion by antonioberetini** [fastlane/fastlane#18566] [Ideas]
Giving that Microsoft is fully embracing progressive web apps and now with the new Edge WebView along with PWA Builder CLI it would be a great idea to add support for Windows store submissions in Fastlane. My company is building hybrid apps and we use Fastlane do deploy React applications wrapped in WebView and capacitor.js to Android and iOS. Since our apps are fully PWA enabled it would be a powerful addition if we can package and auto deploy apps via Fastlane for Windows as well.

I am happy to contribute financially to get this implemented if anyone is interested to work on this.

https://github.com/pwa-builder/PWABuilder-CLI

**Comment by Maurice412** [comment]
I don’t get the point, it’s nice that they have a shared account for open source projects so that not everyone that is just a student has to pay to publish?


[TargetPayandBenefits](https://www.targetpayandbenefits.win/)

**Comment by max-ott** [comment]
I don't think this initiative will come from the core team. Our primary focus is around iOS apps. Android already brought in a lot of friction and works. Feel free to hire a freelancer or similar to this project. More than happy to help him/her to integrate it into `fastlane` if wanted.
